invenio/base/scripts/database.py

```diff
@@ -85,7 +85,7 @@ def init(user='root', password='', yes_i_know=False):
             # Create user and grant access to database.
             (cmd_prefix + '-e "GRANT ALL PRIVILEGES ON '
              '{CFG_DATABASE_NAME}.* TO {CFG_DATABASE_USER}@localhost '
-             'IDENTIFIED BY {CFG_DATABASE_PASS}"'),
+             'IDENTIFIED BY \'{CFG_DATABASE_PASS}\'"'),
```

Comment:
@eamonnmag actually this highlights the issue of proper escaping of the password. What if the password has a quotation sign. You should probably protect the `CFG_DATABASE_PASS` variable within args, so that any single quote is properly escaped.